.github/workflows/update-snapcraft.yaml

```diff
@@ -45,22 +52,46 @@ jobs:
           cd ${{ env.SNAP_REPO_PATH }} && git diff --exit-code snap/snapcraft.yaml \
               && echo "create_pr=0" >> $GITHUB_OUTPUT \
               || echo "create_pr=1" >> $GITHUB_OUTPUT
-      - name: Create Pull Request
+      - name: Upload snapcraft.yaml to github artifact
+        uses: actions/upload-artifact@v4
         if: ${{ steps.update_snapcraft.outputs.create_pr == 1 }}
-        uses: peter-evans/create-pull-request@v6
-        id: create_pr
         with:
-          token: ${{ secrets.PR_PAT }}
-          path: ${{ env.SNAP_REPO_PATH }}
-          title: "release(${{ inputs.openstack-release }}): update snapcraft.yaml"
+          name: updated_snapcraft_yaml
```

Comment:
Can we change this name to contain OpenStack release? Actually checking following workflow and I think this is even required, since it's run by one job and upload artifact will fail with message "file with name updated_snapcraft_yaml" already exist.

Reply:
Updated